app/authentication/session_storage.py

```diff
@@ -1,144 +1,98 @@
-from uuid import uuid4
-
-from flask import session
 from structlog import get_logger
-
 from app.data_model.models import EQSession, db
+from app.data_model.session_data import SessionData
+from app.helpers.cookie_helper import get_eq_session_id
+import simplejson as json
 
-USER_ID = 'user_id'
-USER_IK = 'user_ik'
-SURVEY_COMPLETED_METADATA = 'survey_completed_metadata'
-EQ_SESSION_ID = 'eq-session-id'
 
 logger = get_logger()
 
 
 class SessionStorage:
 
-    @staticmethod
-    def store_user_id(user_id):
+    def store_session_variables(self, user_id, metadata, eq_session_id):
         """
         Create a new eq_session_id and associate it with the user_id specified
         """
-        eq_session_id = str(uuid4())
-        session[EQ_SESSION_ID] = eq_session_id
-        eq_session = EQSession(eq_session_id, user_id)
+        session_data = self._create_session_data_from_meta(metadata)
+        eq_session = EQSession(eq_session_id, user_id, json.dumps(session_data.__dict__))
```

Comment:
i think the best practice is not to use `__dict__` but to use `vars(session_data)` instead